README.md

```diff
@@ -197,7 +197,7 @@ Or you can use `NVActivityIndicatorPresenter` to display UI blocker anywhere.
 Start animating.
 
 ```swift
-let activityData = ActivityData()
+let activityData = ActivityData(controller: ViewController())
```

Comment:
I'd suggest `controller: controller` instead, since this might confuse readers where they need to pass in the view controller reference or create a new view controller.